packages/dx-react-scheduler/docs/guides/editing.md

```diff
@@ -0,0 +1,64 @@
+# React Scheduler - Editing
+
+The Scheduler supports editing features including creating, updating and deleting appointments. Use the corresponding plugins to manage the editing state and perform editing operations programmatically or via the UI controls. The editing state contains information about appointment currently being edited, changes applied to a particular appointment, and appointment that has been deleted and created but not yet committed. Once a user accepts a appointment addition or deletion, or changes made to an appointment (clicking the Save or Delete button), the Scheduler fires the `EditingState` plugin's `onCommitChanges` event and resets the appointment's editing state.
+
+## User Interactions
+
+The React Scheduler provide a functionality to manage the data source via user interface. End-user can add, edit and delete appointments by built-in UI controls. Rules below introduce to user's editing manipulations.
```

Comment:
> The React Scheduler provide...
The React Scheduler provides

Reply:
>  a functionality to manage the data source

It usually means managing mappings, endpoint urls etc. Maybe better just 'a functionality to manage appointments'